tsweb/promvarz/promvarz.go

```diff
@@ -28,7 +28,7 @@ func Handler(w http.ResponseWriter, r *http.Request) {
 // gatherNativePrometheusMetrics writes metrics from the default
 // metric registry in text format.
 func gatherNativePrometheusMetrics(w http.ResponseWriter) error {
-	enc := expfmt.NewEncoder(w, expfmt.FmtText)
+	enc := expfmt.NewEncoder(w, expfmt.NewFormat(expfmt.TypeTextPlain))
```

Comment:
is this related to this PR?

Reply:
Yes. This PR required an update of prometheus/common, and the API surface changed slightly.